src/test/python/test_tt.py

```diff
@@ -53,52 +53,52 @@ def test_convert(self):
     # This gets processed as 2 args being passed into
     # the main function
     #
-    tt.main("convert -i src/test/resources/ttml/body_only.ttml -o build/body_only.out.ttml".split())
+    tt.main("convert -i src/test/resources/ttml/body_only.ttml -o build/body_only.out.ttml --config_file build/config.json".split())
```

Comment:
The config file should be under `src/test/resources/config_files` since `build` is not committed.

Reply:
The config_file should have a descriptive name, e.g. `unit_test_cfg.json`